terraform/aspen_batch.tf

```diff
@@ -69,7 +69,7 @@ resource "aws_batch_compute_environment" "aspen-batch-compute-environment" {
   compute_environment_name_prefix = "aspen-batch-"
 
   compute_resources {
-    instance_role = aws_iam_instance_profile.aspen-batch-ecs-instance-role.arn
+    instance_role = aws_iam_instance_profile.batch-ecs-instance-role.arn
```

Comment:
did you think adding the aspen prefix is too redundant?

Reply:
uh, I don't have a strong opinion on that.